hokusai/test.yml

```diff
@@ -1,7 +1,7 @@
 version: "2"
 services:
   force:
-    command: yarn test
+    command: yarn test:smoke
```

Comment:
I think we'd want to add this as an additional test job in CI, similar to how the `acceptance` step runs here: https://github.com/artsy/force/blob/master/.circleci/config.yml#L11

Reply:
Yes, definitely 👍 . I am just squatting on the existing job while this is WIP so that I could focus on the Cypress stuff instead of the Circle config.